docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Graph-Backed Assets | Dagster
+description: Defining a software-defined asset with multiple discrete computations combined in a graph.
+---
+
+# Graph-Backed Assets
+
+A graph-backed asset is a software-defined asset whose computations are separated into multiple [ops](/concepts/ops-jobs-graphs/ops) that are combined to build a [graph](/concepts/ops-jobs-graphs/graphs). A graph-backed asset can output multiple assets.
+
+---
+
+## Relevant APIs
+
+| Name                                                       | Description                                   |
+| ---------------------------------------------------------- | --------------------------------------------- |
+| <PyObject object="AssetsDefinition" method="from_graph" /> | Constructs an asset given a graph definition. |
+
+---
+
+## Overview
+
+[Basic software-defined assets](software-defined-assets#a-basic-software-defined-asset) are computed using a single op. If generating an asset involves multiple discrete computations, you can use graph-backed assets by separating each computation into an op and building a graph to combine your computations. This allows you to launch re-executions of runs at the op boundaries but doesn't require you to link each intermediate value to an asset in persistent storage.
+
+Graph-backed assets are useful if you have an existing graph that produces and consumes assets. Wrapping your graph inside a software-defined asset gives you all the benefits of software-defined assets — like cross-job lineage — without requiring you to change the code inside your graph.
+
+Additionally, graph-backed assets allow you to reuse an existing op across other graph-backed assets and jobs, or within the same graph.
+
+## Defining graph-backed assets
```

Comment:
I've also been adding horizontal rules before any H2-level headings to visually break up the page - can we please do that here?

```suggestion
---

## Defining graph-backed assets
```